data/Debian.yaml

```
@@ -7,5 +7,5 @@ rundeck::repo_config:
     release: any
     repos: main
     key:
-      name: rundeck
+      name: rundeck.gpg
```

`curl --location https://packages.rundeck.com/pagerduty/rundeck/gpgkey` returns an ASCII-armored key, so the name has to end with `.asc`.

```suggestion
      name: rundeck.asc
```